changelog/upgrade-notes.md

```diff
@@ -2,6 +2,10 @@
 
 ### Next release
 
+#### Renamed `ol/source/TileUTFGrid` to `ol/source/UTFGrid`
+
+The module name is now named `ol/source/UTFGrid`.
```

Comment:
"The module name is now `ol/source/UTFGrid` (`ol.source.UTFGrid` in the full build).

Reply:
changed